Content.Shared/Anomaly/SharedAnomalySystem.cs

```diff
@@ -109,9 +109,9 @@ public void DoAnomalyPulse(EntityUid uid, AnomalyComponent? component = null)
         var pulse = EnsureComp<AnomalyPulsingComponent>(uid);
         pulse.EndTime  = Timing.CurTime + pulse.PulseDuration;
         Appearance.SetData(uid, AnomalyVisuals.IsPulsing, true);
-
-        var ev = new AnomalyPulseEvent(component.Stability, component.Severity);
-        RaiseLocalEvent(uid, ref ev);
+        
+        var ev = new AnomalyPulseEvent(uid, component.Stability, component.Severity);
+        RaiseLocalEvent(uid, ref ev, true);
```

Comment:
dont need to broadcast these once you do synchronized component

Reply:
then by what logic are the stability and severity change invents a broodcast, but ripple and supercrit are not?

Reply:
check what subscribes to them and see, if they do use it leave it alone but still handle it on the component for synchronizing